scrapy/contrib/loader/processor.py

```diff
@@ -3,10 +3,14 @@
 
 See documentation in docs/topics/loaders.rst
 """
+import locale
+import re
+import string
 
 from scrapy.utils.misc import arg_to_iter
 from scrapy.utils.datatypes import MergeDict
-from .common import wrap_loader_context
+from .common import wrap_loader_context, purge_chars  #todo relative it
```

Comment:
oops, forgot to remove this. As in testing with `__name__ == '__main__'` relative imports tend to break so I had changed it for testing stuff, then unchanged it and forgot to get rid of the comment.